src/applications/vaos/containers/TypeOfCarePage.jsx

```diff
@@ -164,12 +138,14 @@ function mapStateToProps(state) {
   const formInfo = getFormPageInfo(state, pageKey);
   const newAppointment = getNewAppointment(state);
   const address = selectVet360ResidentialAddress(state);
+  const clickedUpdateAddressButton = newAppointment.clickedUpdateAddressButton;
```

Comment:
I don't think you need this line

Reply:
I think i added this const to watch its value during local testing. Looks like I forget to remove it. Thanks for spotting it.